app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarMenuControllerTest.kt

```diff
@@ -124,6 +125,95 @@ class DefaultBrowserToolbarMenuControllerTest {
         unmockkObject(FenixSnackbar.Companion)
     }
 
+    // TODO: These can be removed for https://github.com/mozilla-mobile/fenix/issues/17870
+    // todo === Start ===
+    @Test
+    fun handleToolbarBookmarkPressWithReaderModeInactive() = runBlockingTest {
+        val item = ToolbarMenu.Item.Bookmark
+        val title = "Mozilla"
+        val readerUrl = "moz-extension://1234"
+        val readerTab = createTab(
+            url = readerUrl,
+            readerState = ReaderState(active = false, activeUrl = "https://1234.org"),
+            title = title
+        )
+        browserStore =
+            BrowserStore(BrowserState(tabs = listOf(readerTab), selectedTabId = readerTab.id))
+        every { currentSession.id } returns readerTab.id
+        every { currentSession.title } returns title
+        every { currentSession.url } returns "https://mozilla.org"
+
+        val controller = createController(scope = this)
+        controller.handleToolbarItemInteraction(item)
+
+        verify { metrics.track(Event.BrowserMenuItemTapped(Event.BrowserMenuItemTapped.Item.BOOKMARK)) }
+        verify { bookmarkTapped("https://mozilla.org", title) }
+    }
+
+    @Test
+    fun handleToolbarBookmarkPressWithReaderModeActive() = runBlockingTest {
```

Comment:
There's been varied use of this in Fenix, but I was once told that naming tests with the back apostrophe and the IF THEN WHEN mindset. So fun `Toolbar Bookmark Press With Readermode Inactive Results in Metrics Being Tracked` where the individual can see what the test is supposed to verify from the title alone.

Reply:
Great idea! I'll change this 👍